pkg/testcase/selinux.go

```diff
@@ -209,42 +153,35 @@ func selectSelinuxPolicy(product, osType string) cmdCtx {
 	}
 
 	fmt.Printf("Configuration for %s not found!\n", key)
+
 	return nil
 }
 
 // TestSelinuxSpcT Validate that containers don't run with spc_t
 func TestSelinuxSpcT() {
-	cluster := factory.AddCluster(GinkgoT())
+	cluster, _, err := setHelper()
+	Expect(err).NotTo(HaveOccurred())
 
 	for _, serverIP := range cluster.ServerIPs {
-		res, err := shared.RunCommandOnNode("ps auxZ | grep metrics | grep -v grep", serverIP)
-		Expect(err).NotTo(HaveOccurred())
+		// removing err here since this is actually returning exit 1
+		res, _ := shared.RunCommandOnNode("ps auxZ | grep metrics | grep -v grep", serverIP)
 		Expect(res).ShouldNot(ContainSubstring("spc_t"))
 	}
 }
 
 // TestUninstallPolicy Validate that un-installation will remove the rke2-selinux or k3s-selinux policy
 func TestUninstallPolicy() {
-	product, err := shared.GetProduct()
-	//product, err := shared.GetProduct()
-	if err != nil {
-		log.Println(err)
-	}
-	cluster := factory.AddCluster(GinkgoT())
-	var serverUninstallCmd string
-	var agentUninstallCmd string
-	var serverCmd string
+	cluster, product, err := setHelper()
+	Expect(err).NotTo(HaveOccurred())
+
+	serverUninstallCmd := "sudo rke2-uninstall.sh"
```

Comment:
can we extract these into shared func? also this will fail if the path to `*-uninstall.sh` is different... as in some OS's it is different: `/usr/local/bin` vs `/opt/local/bin`

Reply:
why shared if its specific to selinux ?

Reply:
the uninstall can be re-purposed for other testing, like validating the uninstallation itself, hence I asked if it can be moved...